src/main/java/org/ays/auth/security/AysAuthenticationEntryPoint.java

```diff
@@ -37,6 +38,7 @@ public class AysAuthenticationEntryPoint implements AuthenticationEntryPoint {
      * @param authenticationException the {@link AuthenticationException} that occurred during authentication
      * @throws IOException if an I/O error occurs while sending the response
      */
+
```

Comment:
Javadoclar ile metodlar arasına boşluk koyulmaz.

Reply:
PR'a ihtiyacım yok, influncer değiliz ve burası insanların kendilerini tanıtma yeri değil. İsmim bile yok burada, noname request yolluyorum. Laf sokma yarışına girme 👍🏻
Collab'a hatalarımı nezih bir dille bildirdiği için teşekkür ederim. Yanlışlarım olduğunu öğrendim

Reply:
PR: Pull Request